src/main/java/io/cryostat/platform/internal/PodmanPlatformStrategy.java

```diff
@@ -72,18 +86,63 @@ class PodmanPlatformStrategy implements PlatformDetectionStrategy<PodmanPlatform
     public boolean isAvailable() {
         String socketPath = getSocketPath();
         logger.info("Testing {} Availability via {}", getClass().getSimpleName(), socketPath);
-        // TODO check that the service is actually available on the socket using an HTTP request
-        boolean available = fs.isReadable(fs.pathOf(socketPath));
+
+        boolean socketExists = fs.isReadable(fs.pathOf(socketPath));
+        boolean nativeEnabled = vertx.get().isNativeTransportEnabled();
+
+        if (!nativeEnabled && !Epoll.isAvailable()) {
```

Comment:
Just a question here:

`vertx.get().isNativeTransportEnabled()` checks if any native transport picked up by Vertx is supported?

`Epoll.isAvailable()` checks if netty native transport is supported?

Reply:
> vertx.get().isNativeTransportEnabled() checks if any native transport lib picked up by Vertx?

It checks that there are netty native transport libs available and that Vert.x has been configured to use them:

https://github.com/cryostatio/cryostat/blob/dda9672e650c2203217a8247cd468c12e0bf3f61/src/main/java/io/cryostat/net/NetworkModule.java#L169

> Epoll.isAvailable() checks if netty native transport lib is initialized?

Correct, Epoll is netty's native transport on Linux.

> Because of conflicts, netty is not initialized, causing isNativeTransportEnabled to return false -> can't communicate with podman socket?

Pretty close: netty is initialized but it cannot load Epoll. Since that's the native transport for Linux, and required for netty (and therefore the Vert.x WebClient) to talk to Unix domain sockets, it becomes unable to communicate with the Podman API over the socket.